packages/salesforcedx-vscode-core/resources/dark/refresh.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><path d="M13.451 5.609l-.579-.939-1.068.812-.076.094c-.335.415-.927 1.341-1.124 2.876l-.021.165.033.163.071.345c0 1.654-1.346 3-3 3-.795 0-1.545-.311-2.107-.868-.563-.567-.873-1.317-.873-2.111 0-1.431 1.007-2.632 2.351-2.929v2.926s2.528-2.087 2.984-2.461h.012l3.061-2.582-4.919-4.1h-1.137v2.404c-3.429.318-6.121 3.211-6.121 6.721 0 1.809.707 3.508 1.986 4.782 1.277 1.282 2.976 1.988 4.784 1.988 3.722 0 6.75-3.028 6.75-6.75 0-1.245-.349-2.468-1.007-3.536z" fill="#2D2D30"/><path d="M12.6 6.134l-.094.071c-.269.333-.746 1.096-.91 2.375.057.277.092.495.092.545 0 2.206-1.794 4-4 4-1.098 0-2.093-.445-2.817-1.164-.718-.724-1.163-1.718-1.163-2.815 0-2.206 1.794-4 4-4l.351.025v1.85s1.626-1.342 1.631-1.339l1.869-1.577-3.5-2.917v2.218l-.371-.03c-3.176 0-5.75 2.574-5.75 5.75 0 1.593.648 3.034 1.695 4.076 1.042 1.046 2.482 1.694 4.076 1.694 3.176 0 5.75-2.574 5.75-5.75-.001-1.106-.318-2.135-.859-3.012z" fill="#C5C5C5"/></svg>
\ No newline at end of file
+<?xml version="1.0"?><svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 52 52"><path fill="#b5b5b5" d="m46.5 4h-3c-0.8 0-1.5 0.7-1.5 1.5v7c0 0.9-0.5 1.3-1.2 0.7-0.3-0.4-0.6-0.7-1-1-5-5-12-7.1-19.2-5.7-2.5 0.5-4.9 1.5-7 2.9-6.1 4-9.6 10.5-9.7 17.5-0.1 5.4 2 10.8 5.8 14.7 4 4.2 9.4 6.5 15.2 6.5 5.1 0 9.9-1.8 13.7-5 0.7-0.6 0.7-1.6 0.1-2.2l-2.1-2.1c-0.5-0.5-1.4-0.6-2-0.1-3.6 3-8.5 4.2-13.4 3-1.3-0.3-2.6-0.9-3.8-1.6-5.7-3.5-8.4-10.1-6.8-16.7 0.3-1.3 0.9-2.6 1.6-3.8 2.8-4.9 7.7-7.6 12.9-7.6 4 0 7.8 1.6 10.6 4.4 0.5 0.4 0.9 0.9 1.2 1.4 0.3 0.8-0.4 1.2-1.3 1.2h-7c-0.8 0-1.5 0.7-1.5 1.5v3.1c0 0.8 0.6 1.4 1.4 1.4h18.3c0.7 0 1.3-0.6 1.3-1.3v-18.2c-0.1-0.8-0.8-1.5-1.6-1.5z"></path></svg>
```

Comment:
Where did you get these from ?

Reply:
We should use the icons that are in the VS Code source repo so they are exactly the same. We've used them before and already have their licenses and such referenced I believe.